<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Window Detective</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qt_resources.qrc">
    <normaloff>:/Window Detective.ico</normaloff>:/Window Detective.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="margin">
     <number>2</number>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>844</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSpy">
    <property name="title">
     <string>&amp;Inspect</string>
    </property>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="actionViewWindowProperties"/>
    <addaction name="actionSetWindowProperties"/>
    <addaction name="separator"/>
    <addaction name="actionExpandAll"/>
    <addaction name="separator"/>
    <addaction name="actionSetStyles"/>
    <addaction name="separator"/>
    <addaction name="actionFlashWindow"/>
    <addaction name="actionShowWindow"/>
    <addaction name="actionHideWindow"/>
    <addaction name="separator"/>
    <addaction name="actionCloseWindow"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionRefresh"/>
   </widget>
   <widget class="QMenu" name="menuProcess">
    <property name="title">
     <string>Process</string>
    </property>
    <addaction name="actionExpandAll"/>
   </widget>
   <addaction name="menuSpy"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuProcess"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="treeDock">
   <property name="whatsThis">
    <string>Lists all windows in the system, sorted either by relationship to the desktop window or by owner process</string>
   </property>
   <property name="windowTitle">
    <string>Window Hierarchy</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QTabWidget" name="treeTabs">
       <property name="tabPosition">
        <enum>QTabWidget::South</enum>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="desktopWindowTab">
        <attribute name="icon">
         <iconset resource="../qt_resources.qrc">
          <normaloff>:/img/window.png</normaloff>:/img/window.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Window</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTreeWidget" name="desktopWindowTree">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <column>
            <property name="text">
             <string notr="true">Window</string>
            </property>
            <property name="whatsThis">
             <string>Displays the name of the window class</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="processWindowTab">
        <attribute name="icon">
         <iconset resource="../qt_resources.qrc">
          <normaloff>:/img/process.png</normaloff>:/img/process.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Process</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTreeWidget" name="processWindowTree">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <column>
            <property name="text">
             <string notr="true">Window/Process</string>
            </property>
            <property name="whatsThis">
             <string>Displays the name of the window class or process</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="pickerToolBar">
   <property name="windowTitle">
    <string>Picker ToolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Main ToolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFind"/>
   <addaction name="actionRefresh"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
  </widget>
  <widget class="QDockWidget" name="statusDock">
   <property name="whatsThis">
    <string>Displays any status messages or window events as they happen</string>
   </property>
   <property name="windowTitle">
    <string>Logs</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="LogWidget" name="messageLogList">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="cornerButtonEnabled">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>20</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>20</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="../qt_resources.qrc">
     <normaloff>:/img/preferences.png</normaloff>:/img/preferences.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../qt_resources.qrc">
     <normaloff>:/img/help.png</normaloff>:/img/help.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Help Contents</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About Window Detective</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="../qt_resources.qrc">
     <normaloff>:/img/find.png</normaloff>:/img/find.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Find...</string>
   </property>
   <property name="toolTip">
    <string>Find a window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="../qt_resources.qrc">
     <normaloff>:/img/refresh.png</normaloff>:/img/refresh.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh all window</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionViewWindowProperties">
   <property name="text">
    <string>View &amp;Properties</string>
   </property>
   <property name="toolTip">
    <string>View the window's properties</string>
   </property>
  </action>
  <action name="actionSetWindowProperties">
   <property name="text">
    <string>Se&amp;t Properties...</string>
   </property>
   <property name="toolTip">
    <string>Set the window's properties</string>
   </property>
  </action>
  <action name="actionShowWindow">
   <property name="text">
    <string>&amp;Show</string>
   </property>
   <property name="toolTip">
    <string>Show the window and bring it to the front</string>
   </property>
  </action>
  <action name="actionHideWindow">
   <property name="text">
    <string>&amp;Hide</string>
   </property>
   <property name="toolTip">
    <string>Hides the window</string>
   </property>
  </action>
  <action name="actionFlashWindow">
   <property name="text">
    <string>Flash</string>
   </property>
   <property name="toolTip">
    <string>Flashes the window for a short time</string>
   </property>
  </action>
  <action name="actionExpandAll">
   <property name="text">
    <string>Expand All</string>
   </property>
   <property name="toolTip">
    <string>Expands all nodes of the tree</string>
   </property>
  </action>
  <action name="actionSetStyles">
   <property name="text">
    <string>Edit Window Styles</string>
   </property>
   <property name="toolTip">
    <string>Set the window styles</string>
   </property>
  </action>
  <action name="actionCloseWindow">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LogWidget</class>
   <extends>QTableWidget</extends>
   <header>ui/custom_widgets/LogWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qt_resources.qrc"/>
 </resources>
 <connections/>
</ui>
